pkg/roachprod/install/expander.go

```diff
@@ -119,14 +119,22 @@ func (e *expander) maybeExpandPgURL(
 	}
 
 	if e.pgURLs == nil {
+		e.pgURLs = make(map[string]map[Node]string)
```

Comment:
nit: could you modify the docstring of this func to indicate the new pattern is `{pgurl:<nodeSpec>:tenantName}`?